docs/user_guide/concepts/control_plane/domains.rst

```diff
@@ -10,4 +10,4 @@ deployment environments.
 
 For example: We develop and deploy Flyte workflows in development, staging, and production. We configure Flyte domains with those names, and specify lower resource limits on the development and staging domains than production domains. 
 
-We also use domains to disable launch plans and schedules from development and staging domains, since those features are typically meant for production deployments.
\ No newline at end of file
+We also use domains to disable launch plans and schedules from development and staging domains, since those features are typically meant for production deployments. Domains are fixed and unique at the global level.
```

Comment:
> Domains are fixed and unique at the global level.

It feels a bit disconnected, I think it would go better at the intro of the page, like

"Domains are fixed and unique at the global level, and provide an abstraction to isolate resources and feature configuration for different deployment environments"

Reply:
Thank you very much for the detailed suggestion 🙌❤️

Reply:
I have improved the positioning of the text as suggested by you @davidmirror-ops, please have a look at the updated Pull Request 🙌🚀